src/components/CardHead.js

```diff
@@ -19,24 +28,74 @@ export default function CardHead({
   handleDelete,
   id,
   pathname,
+  setProfiles,
+  profiles,
 }) {
   const { on, toggle } = useToggle(false)
```

Comment:
What is 'on' and what is 'toggle'. Maybee you can specify this more precise in the naming.

Reply:
Hmm not sure how to specify it as it toggles on click and if I name it after my components thats more confusing.